pyproject.toml

```diff
@@ -7,6 +7,8 @@ filterwarnings = [
     "ignore:^pkg_resources is deprecated as an API:DeprecationWarning:pkg_resources",
     "ignore:^pkg_resources is deprecated as an API:DeprecationWarning:pyramid",
     "ignore:^Deprecated call to .pkg_resources\\.declare_namespace\\('.*'\\).\\.:DeprecationWarning:pkg_resources",
+    # https://github.com/webpy/webpy/issues/732
+    "ignore:^\\'cgi\\' is deprecated and slated for removal in Python 3\\.13:DeprecationWarning:webob",
```

Comment:
We removed our usage of this library but our dependencies still use it.

This blocks the upgrade to 3.13 I suppose but we have plenty of time to worry about that.